tests/conftest.py

```diff
@@ -64,6 +64,17 @@ def rabbitmq_broker():
     broker.close()
 
 
+@pytest.fixture()
+def rabbitmq_broker_state():
```

Comment:
I don't think you need to to add this, you can, use the classic `rabbitmq_broker`